### PR TITLE
Weather in Stockholm - Ingela Löfgren

### PR DESCRIPTION
**Pull request by IngelaL**
Link to my deployed project: https://weatherinstockholm.netlify.app/

My reflections on how this week's project turned out: I would have made more on this project but unfortunatley other things (personal issues) came in between.

Things I'd like to have clarified or explained in more detail: not really.

### CODE REVIEW

**Review by HenrikeW — CHANGES_REQUESTED** [review]
You wrote really clean code that was easy to read and follow. Very structured and straight forward, great! :) I commented some smaller issues regarding semantics or naming conventions that you can have a look at if there is time. Otherwise, I think you should add some more responsiveness to fulfill the blue requirements. But still, with regards to the little time you had and your personal situation right now, you did a really good job! Thumbs up 👍

**Comment by IngelaL** [comment]
Great review Henrike. It gave me a lot to reflect on and to learn from. I have now made the site more responsive which was a change request.  Thank you!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
```
Comment (EmOlsz) [inline index.html:6]: Let's not forget about `./` in path to file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <div id="sunset"></div>
+</div>
+
+<div class="forecast">
+    <div id="forecastDayAndTemp"></div>
+</div>
+    
+</section>
+
+<script src="script.js"></script>
```
Comment (EmOlsz) [inline index.html:28]: `./`

File: script.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+
+//TODAY'S WEATHER: CITY, TEMPERATURE, SUNSET AND SUNRISE
+
+const city = document.getElementById("city");
+const weatherToday = document.getElementById("weatherToday");
+const temperature = document.getElementById("temperature");
+const sunrise = document.getElementById("sunrise");
+const sunset = document.getElementById("sunset");
+
+const weatherApi = "https://api.openweathermap.org/data/2.5/weather?q=Stockholm,Sweden&units=metric&APPID=1c055fa282f5d9954e97fd78b7cd3c03";
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply script.js:10]: 👍

File: script.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        
+    filteredForecast.forEach(item => {
+    const date = new Date(item.dt * 1000)
+    const dayName = date.toLocaleDateString("en-US", { 
+        weekday: "short" 
+  });
+        
+    const dayTemperature = item.main.temp.toFixed(1);
+
+    forecastDayAndTemp.innerHTML += `<h5>${dayName}: ${dayTemperature} °C</h5>`;
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply script.js:53]: Definitely, good catch Henrike :)

File: style.css [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+.weather-app {
+  font-family: 'Montserrat', sans-serif;
+  border: double black;
+  max-width: 400px;
+  margin-left: auto;
+  margin-right: auto;
+  background: linear-gradient(to top, #2980b9, #6dd5fa, #ffffff);
+}
+
+h1 {
```
Comment (EmOlsz) [inline style.css:15]: You could use more CSS classes, consider it for the next time :)

File: style.css [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+.forecast {
+  margin-bottom: 90px;
+}
+
+
+@media (min-width:668px) and (max-width: 1023px) {
+
+body {
```
Comment (EmOlsz) [inline style.css:56]: Wrong indentation

File: style.css [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+  background-image: url(/weatherBackgroundTablet.jpg);
```
Comment (EmOlsz) [inline style.css:57]: Let's try to wrap url inside quotes - some browsers might have problem with reading url without them :

```
background-image: url("/weatherBackgroundTablet.jpg");
```

File: style.css [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+  background-size: cover;
+}
+
+.weather-app {
+  max-width: 700px;
+  background: linear-gradient(to right, #e8cbc0, #636fa4);
+}
+}
+
+@media (min-width:1024px) {
```
Comment (EmOlsz) [inline style.css:67]: Wrong indentation inside this `@media` as well

**Review by EmOlsz — APPROVED** [review]
Nice project Ingela, good job! You've done basic requirements in a very nice, clean way. Congratulations ;)
Take a look on my comments, they might help you improve even more!

**Pull request closed without merging**